### DISCUSSION TITLE
Otel Warning -  Add amount must be non-negative on Counter

### DISCUSSION THREAD

**Discussion by wolvery** [apache/airflow#42501] [Q&A]
Hey, 
I'm having this warning using Airflow 2.9.0 on Kubernetes with OpenTelemetry.
Based on the documentation, it seems we got a negative value. 
Should this metric be a counter?

**Comment by jakubmatyszewski** [comment]
This problem is referenced here: https://github.com/apache/airflow/blame/538e8d37a81872db58c10e56bbe54dacbc5ff08c/airflow-core/src/airflow/metrics/otel_logger.py#L55-L68
